src/ray/raylet/worker.h

```diff
@@ -205,6 +208,8 @@ class Worker : public WorkerInterface {
     lifetime_allocated_instances_ = allocated_instances;
   };
 
+  const ActorID &GetRootDetachedActorId() const { return root_detached_actor_id_; }
```

Comment:
what does root detached actor mean?

Reply:
If a task or actor is created by a detached actor (directly or transitively) then its root is the detached actor otherwise it's the driver.